File: VB/WindowsFormsApplication32/MyTabbedMdiManager/ActiveFilesButtonClickEventArgs.vb
Imports Microsoft.VisualBasic
Imports System
Imports DevExpress.XtraTab.Buttons

Namespace CustomTabbedMdiManager
	Public Class ActiveFilesButtonClickEventArgs
		Inherits EventArgs
		Private clickedButtonInfo As TabButtonInfo

		Public Sub New(ByVal button As TabButtonInfo)
			clickedButtonInfo = button
		End Sub

		Public ReadOnly Property ButtonInfo() As TabButtonInfo
			Get
				Return clickedButtonInfo
			End Get
		End Property
	End Class
End Namespace

File: CS/WindowsFormsApplication32/MyTabbedMdiManager/ActiveFilesButtonClickEventArgs.cs
using System;
using DevExpress.XtraTab.Buttons;

namespace CustomTabbedMdiManager
{
	public class ActiveFilesButtonClickEventArgs : EventArgs
	{
		private TabButtonInfo clickedButtonInfo;

		public ActiveFilesButtonClickEventArgs(TabButtonInfo button)
		{
			clickedButtonInfo = button;
		}

		public TabButtonInfo ButtonInfo
		{
			get
			{
				return clickedButtonInfo;
			}
		}
	}
}
